wp-config-pantheon.php

```diff
@@ -88,3 +88,15 @@
             break;
     }
 }
+
+/**
+ * Resolves valhalla performance issue
+ * 
+ * If the warning below occurs, check if this constant is being defined 
+ * again in code that runs after this:
+ * 
+ * PHP Warning:  Constant FS_METHOD already defined in...
```

Comment:
Try it with an actual site so that we can be specific about where the error message appears and what the symptoms are. Then, advise the reader to remove other occurrences of `define('FS_METHOD', ...)` from their code.

Reply:
When you say "where the error message appears", I assume you mean the one when the constant is defined twice, rather than the valhalla issue itself...

I will try defining it in `wp-config.php` after `wp-config-pantheon.php` is already loaded

Reply:
Yes, that is what I mean.